cli/transmission-cli.1

```diff
@@ -74,17 +74,41 @@ Enable portmapping via NAT-PMP or UPnP
 Disable portmapping
 .It Fl p -port Ar port
 Set the port to listen for incoming peers. (Default: 51413)
-.It Fl t -tos Ar tos
-Set the peer socket TOS for local router-based traffic shaping.
-Valid values are
+.It Fl -tos Ar tos
+.nf
+Use a ToS or DSCP name to set the peer socket service type for local router-based traffic shaping. (Default: le)
+Valid values are:
+.fi
 .Pp
+.Dq af11 ,
+.Dq af12 ,
+.Dq af13 ,
+.Dq af21 ,
+.Dq af22 ,
+.Dq af23 ,
+.Dq af31 ,
+.Dq af32 ,
+.Dq af33 ,
+.Dq af41 ,
+.Dq af42 ,
+.Dq af43 ,
+.Dq cs0 ,
+.Dq cs1 ,
+.Dq cs2 ,
+.Dq cs3 ,
+.Dq cs4 ,
+.Dq cs5 ,
+.Dq cs6 ,
+.Dq cs7 ,
+.Dq ef ,
+.Dq le ,
 .Dq default ,
```

Comment:
I'm not sure if the old IPTOS values should be documented since DSCP is not backward compatible with IPToS therefore they shouldn't be used. In the rare case some equipment insists on IPToS they could instead set its value by directly inputting the decimal code for it.

Reply:
I updated this in 334ac3a0bbd21976a7578e1c03424d9f85b8acb0 to still list them, since they are supported in Transmission 3.0, but to denote them as deprecated and slated for future removal.

Reply:
That's actually a bug in libtorrent, they're using the "QBone scavenger service" DSCP which is defined in [RFC 8622](https://datatracker.ietf.org/doc/html/rfc8622) which has DSCP _codepoint_ 0x01 (accounting for the ECN bits it results in 0x04). The incorrect 0x20 value is obtained if you count the bits backward.